File: Recommender/src/main/java/cn/edu/ahtcm/caseclass/recs/ProductRecs.scala
package cn.edu.ahtcm.caseclass.recs

/**
 * 定义商品相似度列表
 *
 * @param productId     商品 ID
 * @param recsProductId 推荐商品 ID
 * @param score         评分
 */
case class ProductRecs(productId: Int, recsProductId: Int, score: Double)

File: Mall/src/main/java/cn/edu/ahtcm/mall/bean/ProductRecs.java
package cn.edu.ahtcm.mall.bean;

public class ProductRecs {

    private int productId;
    private int recsProductId;
    private Double score;

    public int getProductId() {
        return productId;
    }

    public void setProductId(int productId) {
        this.productId = productId;
    }

    public int getRecsProductId() {
        return recsProductId;
    }

    public void setRecsProductId(int recsProductId) {
        this.recsProductId = recsProductId;
    }

    public Double getScore() {
        return score;
    }

    public void setScore(Double score) {
        this.score = score;
    }
}
